pkg/services/provisioning/notifiers/types.go

```diff
@@ -1,30 +1,61 @@
 package notifiers
 
-import "github.com/grafana/grafana/pkg/components/simplejson"
```

Comment:
Small refactor with the types here to align it with other provisioning types and makes the change to Value much more contained. Basically we have `ConfigVx` type that can be versioned and yaml file is unmarshalled into it. Afterwards there is `Config` type that all `ConfigVx` are mapped into which separates the multiple possible config versions from internal representation. This also means that changes to the config unmarshalling are only done here and does not permeate other parts of the code that needs to work with the data.